configuration.packages.json

```diff
@@ -67,7 +67,7 @@
     },
     "EdFi.Db.Deploy": {
       "PackageName": "EdFi.Suite3.Db.Deploy",
-      "PackageVersion": "4.0.31",
+      "PackageVersion": "4.0.37",
```

Comment:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/actions/runs/7463341922/job/20307788781#step:15:20 

4.0.38 is created from your branch after your change . Please update  that